### PR TITLE
add workshop content

### PR DESCRIPTION
**Pull request by TomasTomecek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+      target_image:
+        name: tmux-top-tests
+        environment:
+          A: B
+  tasks: []
+```
+
+Our task now is to clone the upstream repo -
+https://github.com/TomasTomecek/tmux-top/, and run the tests using `make tests.
```
Comment (jwerak) [inline README.md]: just a little thing, but this line is missing ending `

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Building container images with Ansible
```
Comment (jwerak) [inline README.md:1]: Would be great to have lint to ansible-bender with small about line to get quick context when I land on this page.

**Comment by TomasTomecek** [comment]
I'm done now, please review.

I'll do some minor tweaks likely on Thu while on train :D

**Comment by rwngwn** [comment]
Thank you! Sorry for late merging :)

**Pull request merged**